src/our-work/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: layouts/default.njk
+title: Our Work
```

Comment:
To simplify things, we can rename "product and services" to "Our Work" (which I like better as a title), and tweak the wording a little.

Reply:
Yeah I like the idea of putting it all here